test/isotest.js

```diff
@@ -29,7 +29,7 @@ function isReactClass(Component) {
  * @param skipList    {string[]}                             array of component names to skip
  */
 module.exports = function generateIsomorphicTests(Components, props, children, skipList = []) {
-    Object.keys(Components).forEach((componentName) => {
+    Object.keys(Components).sort().forEach((componentName) => {
```

Comment:
from commit message:

> Sort isotest components so the list is easier to scan

makes it easier to immediately identify if a component is missing from the list